src/NBitcoin/Block.cs

```diff
@@ -30,14 +30,13 @@ public MerkleNode GetMerkleRoot()
         public Block()
         {
             this.header = new BlockHeader();
-            this.SetNull();
+            this.transactions.Clear();
```

Comment:
This should probably also have the `Obsolete` attribute

Reply:
OK will do!

Reply:
Ah I see it has already ;)